### PR TITLE
fix: build endpoint for compat API

### PR DESCRIPTION
**Pull request by matejvasek**
resolves: #9290

Signed-off-by: Matej Vasek <mvasek@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
I have a similar PR, but have not been able to get back to this.  Some of this code is in the master branch of Buildah, now I believe.

File: pkg/api/handlers/compat/images_build.go [hunk 1]
```diff
@@ -205,9 +205,15 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		isolation := buildah.IsolationDefault
 	*/
 	if utils.IsLibpodRequest(r) {
-		//		isolation = buildah.Isolation(query.Isolation)
+		// isolation = parseLibPodIsolation(query.Isolation)
```
Comment (TomSweeneyRedHat) [inline pkg/api/handlers/compat/images_build.go:208]: is this comment of any value?

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, but I'm not sure if we should go with this or @rhatdan's WIP?

File: pkg/api/handlers/compat/images_build.go (hunk 1 above)
Comment (matejvasek) [reply pkg/api/handlers/compat/images_build.go:208]: If I understand it correctly the comment should be removed at some point see TODO above.

Comment (rhatdan) [reply pkg/api/handlers/compat/images_build.go:208]: Yes I have a PR that is attempting to remove it.
I think we just merge this and I will continue to rework my PRs

**Pull request merged**